src/components/plus-button/plus-button.spec.tsx

```diff
@@ -0,0 +1,34 @@
+import { screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { ButtonSize } from "../../types";
+import { renderWithTheme } from "../../utils/testing-utils/render-with-theme";
+import { PlusButton } from "./plus-button";
+
+const onClickMock = jest.fn();
+
+const callRender = (renderFunc, props = {}) => {
+  return renderFunc(
+    <PlusButton
+      buttonSize={ButtonSize.Small}
+      onClick={onClickMock}
+      {...props} />
+  );
+};
+
+describe("Plus Button", () => {
+  it("Should render small Plus button", () => {
+    const { container } = callRender(renderWithTheme, { children: 'Test Button' });
+    expect(container).toBeInTheDocument();
+
+    userEvent.click(screen.getByText('Test Button'));
+    expect(onClickMock).toHaveBeenCalled();
+  });
+
+  it("Should render Big Plus button", () => {
+    const { container } = callRender(renderWithTheme, { children: 'Test Button', buttonSize: ButtonSize.Big });
+    expect(container).toBeInTheDocument();
```

Comment:
Could you check the actual size? Test cases are same

Reply:
Done